builder/shellParser_test.go

```diff
@@ -0,0 +1,51 @@
+package builder
```

Comment:
not 100% sure what I was thinking at that exact moment :-)  but I probably put it in the builder because shellParser (which is probably not the best name for it) isn't used during the parsing phase - its actually used during the env-var substitution phase.  Perhaps I should have called it "shellWordProcessor" instead - dunno.